enginetest/queries/script_queries.go

```diff
@@ -2884,9 +2888,8 @@ CREATE TABLE tab3 (
 			},
 			{
 				// Test with nested subqueries
-				Skip:     true,
-				Query:    "SELECT GROUP_CONCAT(name ORDER BY (SELECT COUNT(*) FROM complex_test c2 WHERE c2.value > (SELECT MIN(value) FROM complex_test c3 WHERE c3.category = complex_test.category))) FROM complex_test",
-				Expected: []sql.Row{{"Gamma,Alpha,Epsilon,Beta,Delta"}},
+				Query:    "SELECT GROUP_CONCAT(name ORDER BY (SELECT SUM(value) FROM complex_test c2 WHERE c2.value != (SELECT MIN(value) FROM complex_test c3 where c3.id = complex_test.id))) FROM complex_test;",
```

Comment:
The previous test had values with the same `COUNT(*)`, so the final order was still correct.
I changed the test so there are no ties.